server/apps/immich/src/app.module.ts

```diff
@@ -55,7 +46,12 @@ import { AuthGuard } from './middlewares/auth.guard';
     SystemConfigController,
     UserController,
   ],
-  providers: [{ provide: APP_GUARD, useExisting: AuthGuard }, AuthGuard],
+  providers: [
+    //
+    { provide: APP_GUARD, useExisting: AuthGuard },
+    AuthGuard,
+    AppCronJobs,
```

Comment:
What is the difference between having the Module be imported vs. as a provider in this case?

Reply:
Not much, it is just one file, used in one place, with no depending, and not imported anywhere so there's no benefit to having a dedicated module for it.